### ISSUE TITLE
Bad formatting in Summary string

### ISSUE THREAD

**Issue by briankyncl** [briankyncl/aboutthiscomputer#8]
## Background
Summary string, when displayed in the GUI (File > Summary) needs a return between the "Custom:" text and the next header, "Helpdesk:". When no custom free-text is defined, Summary String has normal spacing.

![SnagIt 2020-01-20 13 26 33](https://user-images.githubusercontent.com/5696821/72758736-7e5f7e80-3b88-11ea-99d4-4db1080604fa.png)

## Desired Outcome
Every section in the summary string should be separated by one blank line.

## TL;DR
Section spacing after "Custom:" is insufficient when the custom string has been defined.
